docs/developer-docs/gas-cost.md

```diff
@@ -109,47 +111,45 @@ The amount of reserved cycles depends on how many bytes are allocated and on the
+When a canister is deployed to a local development environment, the canister is deployed to a single node. Cycles charged to locally deployed canisters have a cost that is 1/13th the cost when deployed to a 13-node subnet. In local development environments, cycles can be fabricated using dfx.
 
 #### Mainnet development
 
-On a 13-node subnet, the canister is running on 13 nodes and therefore is replicated 13 times, whereas on a 34-node subnet, the canister is running on 34 nodes and therefore is replicated 34 times. The cost of resources on a 13-node subnet is different than the cost of resources on a 34-node subnet.
+On a 13-node subnet, the canister is running on 13 nodes and is replicated 13 times. On a 34-node subnet, the canister is running on 34 nodes and is replicated 34 times. The cost of resources on a 13-node subnet is different than the cost of resources on a 34-node subnet.
 
-If you intend to deploy canisters on high-replication subnets, your canister should be prepared for an increase in cycles prices with an increase in the subnet's replication factor when the subnet grows over time. For this reason, it is recommended to attach more cycles to a call than the current price for a high-replication subnet of a given size suggests.
+If you intend to deploy canisters on high-replication subnets, your canister should be prepared for an increase in cycles prices. It is recommended to attach more cycles to a call than the current price for a high-replication subnet.
 
 ## Who is responsible for paying cycles?
 
-Canisters are responsible for paying cycles for their own canister creation, compute resources, storage resources, and execution resources. For certain canister calls, however, the canister responsible for paying the cycles may vary.
+Canisters are responsible for paying cycles for their own canister creation, compute resources, storage resources, and execution resources. For certain canister calls, the canister responsible for paying the cycles may vary.
 
-For inter-canister calls and the associated bytes, the canister sending this call is responsible for paying the required cycles. If there is an inter-canister call chain, for example, canister A → canister B → canister C, then canister A pays for the sending request-response pair between canister A and canister B, and canister B pays for sending the request-response pair between canister B and canister C. Additionally, each canister pays for processing that happens locally, meaning canister A pays for processing the response from canister B while canister B pays for processing the request from canister A.
+A canister sending an inter-canister call is responsible for paying the required cycles. In inter-canister call chains, each canister is responsible for the call that it sends. Each canister pays for processing that happens locally. 
```

Comment:
How about structuring such explanations more as bullet lists. I think ingress is most important and should therefore be on top.
* **Ingress messages:** bla, bla
* **Inter-canister calls:** bla, bla
* **Child canisters:** bla, bla

Reply:
Revised to use a bulleted list.